backend/src/server/createServer.ts

```diff
@@ -1,15 +1,12 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
-
 import 'reflect-metadata'
 import 'module-alias/register'
 
 import { ApolloServer } from 'apollo-server-express'
-import express from 'express'
+import express, { Express } from 'express'
```

Comment:
express same as Express? Why do we want to hold express if Express is used?

Reply:
OK I answered myself the question: Express is the object and is needed for ServerDef! And express is also needed!